### ISSUE TITLE
Parse error, cannot open 'net'

### ISSUE THREAD

**Issue by wbyoung** [guymers/babel-plugin-flow-relay-query#18]
This project looks like a really great idea… having trouble trying it out, though. I'm getting:

```
Error: /Users/wbyoung/Code/app/test/components/my-component.js: ENOENT: no such file or directory, open 'net'
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.readFileSync (fs.js:491:33)
    at parseFile (/Users/wbyoung/Code/app/node_modules/babel-plugin-flow-relay-query/lib/utils/parse.js:26:27)
    at Object.ImportDeclaration (/Users/wbyoung/Code/app/node_modules/babel-plugin-flow-relay-query/lib/flowRelayQueryPlugin.js:255:34)
    at NodePath._call (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/path/context.js:105:12)
    at TraversalContext.visitQueue (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitMultiple (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/context.js:103:17)
    at TraversalContext.visit (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/context.js:190:19)
    at Function.traverse.node (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/index.js:114:17)
    at traverse (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/index.js:79:12)
    at NodePath.traverse (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/path/index.js:144:25)
    at PluginPass.Program (/Users/wbyoung/Code/app/node_modules/babel-plugin-flow-relay-query/lib/flowRelayQueryPlugin.js:278:16)
    at newFn (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/visitors.js:276:21)
    at NodePath._call (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/path/context.js:105:12)
    at TraversalContext.visitQueue (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitSingle (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/context.js:192:19)
    at Function.traverse.node (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/index.js:114:17)
    at traverse (/Users/wbyoung/Code/app/node_modules/babel-traverse/lib/index.js:79:12)
    at File.transform (/Users/wbyoung/Code/app/node_modules/babel-core/lib/transformation/file/index.js:548:35)
    at /Users/wbyoung/Code/app/node_modules/babel-core/lib/transformation/pipeline.js:50:19
    at File.wrap (/Users/wbyoung/Code/app/node_modules/babel-core/lib/transformation/file/index.js:564:16)
    at Pipeline.transform (/Users/wbyoung/Code/app/node_modules/babel-core/lib/transformation/pipeline.js:47:17)
    at Object.transformFileSync (/Users/wbyoung/Code/app/node_modules/babel-core/lib/api/node.js:152:10)
    at compile (/Users/wbyoung/Code/app/node_modules/babel-register/lib/node.js:118:20)
    at loader (/Users/wbyoung/Code/app/node_modules/babel-register/lib/node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/wbyoung/Code/app/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at /Users/wbyoung/Code/app/node_modules/mocha/lib/mocha.js:230:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/wbyoung/Code/app/node_modules/mocha/lib/mocha.js:227:14)
    at Mocha.run (/Users/wbyoung/Code/app/node_modules/mocha/lib/mocha.js:495:10)
    at loadAndRun (/Users/wbyoung/Code/app/node_modules/mocha/bin/_mocha:417:22)
    at Object.<anonymous> (/Users/wbyoung/Code/app/node_modules/mocha/bin/_mocha:434:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:423:7)
    at startup (bootstrap_node.js:147:9)
    at bootstrap_node.js:538:3
```

**Comment by guymers** [comment]
I've created a PR to ignore files that don't exist, it is available in 0.10.10-RC1.

Are you able to post the code of `my-component.js` or a minimal subset that demonstrates the error?

**Comment by wbyoung** [comment]
Honestly, I went a different direction and since Relay Modern generated flow types, I don't think I'll end up using this now. Awesome ideas here, though. Thanks for sharing.

I'll close this since I'm not concerned with it, but feel free to reopen if you wish!
